README.md

```diff
@@ -105,7 +105,7 @@ The content of this library is released under the **MIT License** by
 **Andrew Lawson**.<br/> You can find a copy of this license in
 [`LICENSE`][license] or at http://opensource.org/licenses/mit.
 
-[boxes]:       https://vagrantcloud.com
+[boxes]:       https://atlas.hashicorp.com
```

Comment:
You're right!

> Op 12 mei 2016 om 23:33 heeft ergl <notifications@github.com> het volgende geschreven:
> 
> In README.md:
> 
> > @@ -105,7 +105,7 @@ The content of this library is released under the **MIT License** by
> >  **Andrew Lawson**.<br/> You can find a copy of this license in
> >  [`LICENSE`][license] or at http://opensource.org/licenses/mit.
> >  
> > -[boxes]:       https://vagrantcloud.com
> > +[boxes]:       https://atlas.hashicorp.com
> This url seems to point to a login page, maybe https://atlas.hashicorp.com/boxes/search would be better?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>